include/swift/AST/SearchPathOptions.h

```diff
@@ -302,6 +314,16 @@ class SearchPathOptions {
     Lookup.searchPathsDidChange();
   }
 
+  void setCompilerPluginLibraryPaths(
+      std::vector<std::string> NewCompilerPluginLibraryPaths) {
+    CompilerPluginLibraryPaths = NewCompilerPluginLibraryPaths;
```

Comment:
I wonder... instead of having a new kind of search path for compiler plugins, should we instead require that they always be explicitly reference, e.g., from a module or from the command line?

Reply:
This is already an explicit reference, not a search path. The user directly provides the path to a dylib file, e.g. `-load-plugin-library /path/to/libMyMacro.dylib`.  Or are you suggesting something different?

Reply:
Oh, I misread the code, sorry!